code/controllers/configuration/entries/game_options.dm

```diff
@@ -111,7 +111,10 @@
 	config_entry_value = 15
 	min_val = 1
 
-/datum/config_entry/number/latejoin_larva_required_num
-	integer = FALSE
-	min_val = 0
-	config_entry_value = 4
+/datum/config_entry/number/crash/larvapoints_required
```

Comment:
```suggestion
/datum/config_entry/number/crash_larvapoints_required
```
No need for a subtype here. They don't have different variables to justify a new type.